src/nix/flake.cc

```diff
@@ -454,11 +454,6 @@ struct CmdFlakeCheck : FlakeCommand
                 if (v.payload.lambda.fun->hasFormals()
```

Comment:
If we're removing the `prev` check, should we not also remove the one for `final`? Seems odd to check one but not the other, IMHO.

(And I say this as someone who prefers the `final: prev:` naming convention over anything else)

Reply:
@cole-h Feel free to make a PR for that too once we land this one.